src/components/Feedback/controls/LikeControl.tsx

```diff
@@ -22,42 +24,49 @@ type LikeControlProps = {
 const b = block('dc-feedback');
 
 const LikeControl = memo(
-    forwardRef<HTMLButtonElement, LikeControlProps>(({isLiked, view, onClick}, ref) => {
-        const {t} = useTranslation('feedback');
-        const {isVerticalView, popupPosition, controlClassName} = useContext(ControlsLayoutContext);
-        const tooltipText = isLiked ? t('cancel-like-text') : t('like-text');
+    forwardRef<HTMLButtonElement, LikeControlProps>(
+        ({isLiked, isPopupVisible, view, onClick}, ref) => {
+            const {t} = useTranslation('feedback');
+            const {isVerticalView, popupPosition, controlClassName} =
+                useContext(ControlsLayoutContext);
+            const tooltipText = isLiked ? t('cancel-like-text') : t('like-text');
 
-        const Icon = isLiked ? ThumbsUpFill : ThumbsUp;
+            const Icon = isLiked ? ThumbsUpFill : ThumbsUp;
+
+            if (view === FeedbackView.Wide) {
+                return (
+                    <Button
+                        size="xl"
+                        view="normal"
+                        ref={ref}
+                        onClick={onClick}
+                        className={b('control', {view})}
+                    >
+                        <Button.Icon>
+                            <Icon width={20} height={20} />
+                        </Button.Icon>
+                        {t<string>('button-like-text')}
+                    </Button>
+                );
+            }
 
-        if (view === FeedbackView.Wide) {
             return (
-                <Button
-                    size="m"
-                    view="normal"
-                    ref={ref}
+                <Control
+                    size={ControlSizes.XL}
                     onClick={onClick}
-                    className={b('control', {view})}
-                >
-                    <Button.Icon>
-                        <Icon width={14} height={14} />
-                    </Button.Icon>
-                    {t<string>('button-like-text')}
-                </Button>
+                    className={b('control', {view}, controlClassName)}
+                    isVerticalView={isVerticalView}
+                    tooltipText={tooltipText}
+                    ref={ref}
+                    icon={Icon}
+                    popupPosition={popupPosition}
+                    buttonExtraProps={{
+                        'aria-expanded': isPopupVisible,
```

Comment:
Try to rebase on master. There should be no 'expanded' i deleted them in this commit.

https://github.com/diplodoc-platform/components/commit/d74fbd4f899251afcae000b9fca31aec105a1470